info/v1/container.go

```diff
@@ -902,6 +922,9 @@ type ContainerStats struct {
 
 	// Referenced memory
 	ReferencedMemory uint64 `json:"referenced_memory,omitempty"`
+
+	// Resource Control (resctrl) statistics.
```

Comment:
The rest of comments in `ProcessStats` struct do not have dot at the end.

Reply:
Is it wrong?